shared-libs/transitions/test/unit/transitions/registration.js

```diff
@@ -440,9 +484,12 @@ describe('registration', () => {
       sinon.stub(validation, 'validate').callsArgWith(2, null);
       sinon.stub(utils, 'getRegistrations').resolves([]);
 
-      sinon.stub(transitionUtils, 'addUniqueId').callsFake((doc) => {
-        doc.patient_id = patientId;
-        return Promise.resolve();
+      sinon.stub(transitionUtils, 'getUniqueId').resolves(patientId);
+      sinon.stub(config, 'getAll').returns({
+        contact_types: [
+          { id: 'place' },
+          { id: 'person', person: true, parents: ['place'] },
+        ]
```

Comment:
This is stubbed over and over again - is it worth pulling out either the return as a constant, or putting the stub in a beforeEach?

Reply:
Done.